index.html

```diff
@@ -35,12 +35,22 @@
           <li class ="desktopitem" id="mail"> <a href=""><img src="image/Icon - Mail.png" /></a></li>
           <li class="union">
             <button class="dropdown" type="submit">
-              <img src="image/Union.png" />
+              <img src="image/Union.png"  class= "immage"/>
             </button>
           </li>
         </ul>
       </nav>
     </header>
+   <div class="mobileMenu">
+    <ul >
+          <li><a class="menu-items" href="#work-link"> Portfolio</a> </li>
+          <hr class ="line">
+          <li> <a class="menu-items" href="#about-link"> About</a></li>
+          <hr class ="line">
+          <li> <a class="menu-items" href="#contach-link"> Contact </a></li>
+          <hr class ="line">
+        </ul>
+   </div>
```

Comment:
- [ ] Good job adding the ids in the `hrefs` of the links but you didn't add these ids to the appropriate sections. Please add these ids to all 3 sections so clicking on the mobile menu options user will be redirected to the appropriate section. example for the work section `id=work-link`.